yb-voyager/cmd/cutoverToTargetCmd.go

```diff
@@ -16,16 +16,81 @@ limitations under the License.
 package cmd
 
 import (
+	"fmt"
+	"regexp"
+	"strings"
+
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/metadb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/tgtdb"
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/ybversion"
 )
 
 var prepareForFallBack utils.BoolStr
 var useYBgRPCConnector utils.BoolStr
 
+// validateYBVersionForLogicalConnector checks if the YugabyteDB version is greater than minSupportedLogicalConnectorVersion¯
+// Logical connector is only supported in YugabyteDB versions greater than minSupportedLogicalConnectorVersion
+func validateYBVersionForLogicalConnector(tconf *tgtdb.TargetConf) error {
+	if tconf == nil {
+		return fmt.Errorf("target database configuration is not available")
+	}
+
+	// Version should already be stored in metadata from previous import commands
+	if tconf.DBVersion == "" {
+		return fmt.Errorf("YugabyteDB version not found in metadata.")
+	}
+	versionStr := tconf.DBVersion
+
+	// Extract YB version from PostgreSQL version string
+	// Format: PostgreSQL x.x-YB-version-bxxx or version
+	ybVersion, err := extractYBVersion(versionStr)
+	if err != nil {
+		return fmt.Errorf("failed to extract YugabyteDB version from '%s': %w", versionStr, err)
+	}
+
+	// Parse the version
+	currentVersion, err := ybversion.NewYBVersion(ybVersion)
+	if err != nil {
+		return fmt.Errorf("failed to parse current YugabyteDB version '%s': %w.", ybVersion, err)
+	}
+
+	var minSupportedLogicalConnectorVersion *ybversion.YBVersion
+	if currentVersion.ReleaseType() == ybversion.PREVIEW {
+		minSupportedLogicalConnectorVersion = ybversion.V2_25_0_0
```

Comment:
As discussed in standup, it should be 2.29 but since its not released, lets maybe not support any preview version until then?
So, only the stable version 2024.2.4 now?